### PR TITLE
Adding jekyll

### PR DESCRIPTION
**Pull request by burden**
Mostly just the same site but this time powered by Jekyll & friends, along with...
* Words: to explain what we do
* Hyperlink: to send an email

Behind the scenes we also now have contact@utensils.io

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Utensils.io
+http://utensils.io
```
Comment (doomspork) [inline README.md]: https 👍

File: Rakefile [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+msg = "jekyll-namesake"
+
+task :default do
+  msg = "#{msg}\nFeel free to run a task for more instructions."
+  system "echo #{msg}"
+  system "rake --tasks"
+end
+
+desc 'Generate an encoded email address to stop spammers'
```
Comment (doomspork) [inline Rakefile:19]: What's this all about?

**Review by doomspork — CHANGES_REQUESTED** [review]

File: Rakefile (hunk 2 above)
Comment (burden) [reply Rakefile:19]: It generates an encoded email, which is decoded with javascript into the "hear from you" hyperlink. See the readme for destructions.

File: README.md (hunk 1 above)
Comment (burden) [reply README.md]: 👍 @doomspork

File: _config_stage.yml [hunk 3]
```diff
@@ -0,0 +1 @@
+url: "http://stage.burden.cc"
```
Comment (doomspork) [inline _config_stage.yml]: Let's change this.  If we need be, let's use stage.utensils.io.

**Review by doomspork — CHANGES_REQUESTED** [review]
One last change @burden 👍

**Review by doomspork — APPROVED** [review]

**Comment by doomspork** [comment]
💥 

Thank you @burden!

**Pull request merged**